components/templates/src/global_fns/files.rs

```diff
@@ -228,9 +245,52 @@ impl TeraFn for GetFileHash {
     }
 }
 
+#[derive(Debug)]
+pub struct GetHash {}
```

Comment:
you can use a basic fn like https://github.com/Keats/tera/blob/master/src/builtins/functions.rs#L31 no need to impl the Trait explicitly if there's no state

Reply:
Nice! I’ve applied the change.